uw-frame-components/portal/main.js

```diff
@@ -152,6 +153,10 @@ define([
                 defaultThemeGo();
               }
               loadingCompleteSequence();
+            }, function(reason){
+              console.error("We got a error back from groupURL, setting theme to default");
```

Comment:
I'd avoid console log statements in production code!  It's not really [standards compliant](https://developer.mozilla.org/en-US/docs/Web/API/Console/log) in general, and IIRC fails in IE9 in particular.